src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderCursor.cs

```diff
@@ -430,6 +431,7 @@ public LineReader(IMultiStreamSource files, int batchSize, int bufSize, bool has
                     _separators = separators;
                     _files = files;
                     _cref = cref;
+                    _decimalMarker = decimalMarker;
```

Comment:
I don't think it's necessary to add a `_decimalMarker` member to LineReader. The reason why I added _readMultilines and _separators to it on my PR #5125 was because I needed them on the LineReader.MultilineReader I added on that same PR.

I don't expect MultilineReader (or LineReader) to need _decimalMarker, as it doesn't affect how we read the input file. If anything, I'd think it would only be needed on the Parser. So if LineReader doesn't need it, then there's no need to change the GetSomeLines() method either.

Is there any reason of why you added this in here, and that I might be missing? Thanks 😄

Reply:
The reason why I added this here was to see if I could reach this specific `_decimalMarker` from `Conversion.cs` and `DoubleParser`. I'm planning to remove `_decimalMarker` from here if I find it doesn't help. :D